apiserver/facades/agent/meterstatus/meterstatus_test.go

```diff
@@ -35,11 +35,16 @@ type meterStatusSuite struct {
 	unit *state.Unit
 
 	status meterstatus.MeterStatus
+
+	controllerConfigGetter *mocks.MockControllerConfigGetter
 }
 
 func (s *meterStatusSuite) SetUpTest(c *gc.C) {
 	s.ApiServerSuite.SetUpTest(c)
 
+	ctrl := gomock.NewController(c)
+	s.controllerConfigGetter = mocks.NewMockControllerConfigGetter(ctrl)
+
```

Comment:
We can't do this. You have to put these in a `setupMocks` and call it for every method.

I'm starting to think the test suite isn't working for us when we're using mocks.